.github/workflows/build-interop-docker.yml

```
@@ -19,9 +25,14 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: set tag name
+        if: github.event_name == 'tag'
+        # Tagged releases won't be picked up by the interop runner automatically,
+        # but they can be useful when debugging regressions.
+        run: echo "TAG=${{ github.event.release.tag_name }}"
```

I don't think release object is attached to tag push events. We can extract it from ref if relevant instead.

Also, we need to echo to GitHub context file to be able to use the prepared value. Either `GITHUB_ENV` or `GITHUB_OUTPUT`. Generally, I prefer using output because it makes it more explicit where the values are produced and where they are being used.

```suggestion
        run: |
          if [[ $GITHUB_REF == refs/tags/* ]]; then
            echo "tag=${GITHUB_REF#refs/tags/}" | tee -a $GITHUB_OUTPUT;
          else
            echo 'tag=latest' | tee -a $GITHUB_OUTPUT;
          fi
```